sbt-play-ebean/src/main/scala/play/ebean/sbt/PlayEbean.scala

```diff
@@ -25,7 +40,24 @@ object PlayEbean extends AutoPlugin {
 
   override def trigger = noTrigger
 
-  override def projectSettings = inConfig(Compile)(scopedSettings) ++ unscopedSettings
+  override def projectSettings = {
+    val querySettings = Seq(
+      playEbeanQueryGenerate := false,
+      playEbeanQueryEnhance := false,
+      playEbeanQueryDestDirectory := "app",
```

Comment:
I think for simplicity we just don't give the user the ability to generate Finders for now